loading annotations into memory...
Done (t=12.03s)
creating index...
index created!
Traceback (most recent call last):
  File "./lib/train.py", line 209, in <module>
    train() #debug/train
  File "./lib/train.py", line 146, in train
    loss, net = mask_rcnn(X, True, gt_boxes=gt_boxes, gt_classes=gt_classes, gt_masks=gt_masks)
  File "/home/andy/original-mask-rcnn/lib/mask_rcnn.py", line 82, in mask_rcnn
    cls_feats = crop_proposals(crop_feats, cfg.crop_size, proposals, training)
  File "/home/andy/original-mask-rcnn/lib/rpn.py", line 178, in crop_proposals
    out = tf.image.crop_and_resize(feats[i], cur_boxes/cfg.image_size, batch_ind, [crop_size, crop_size])
IndexError: list index out of range

real	0m29.926s
user	0m15.498s
sys	0m1.455s
